app/controllers/homes_controller.rb

```diff
@@ -3,7 +3,9 @@ def show
     if signed_out?
       render "pages/landing"
     else
-      @recommended_items = RecommendedItem.where(user: current_user.with_following)
+      @recommended_items = RecommendedItem.
```

Comment:
Shouldn't we scope this first to the `current_user` somehow for health and safety?

like

```ruby
current_user.recommended_items.newest_first
```
?